scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1560,10 +1602,10 @@ def verifyDBVersion(version):
 if __name__ == '__main__':
     n = len(sys.argv)
 
-    if n != 25:
+    if n != 26:
         print(
-            "Please pass valid 24 arguments <platform_db-name> <platform_host> <oes-db-name> <oes-db-host> <autopilot-db-name> <autopilot-db-host> <audit_db-name> <audit-db-host> <visibility_db-name> <visibility-db-host> "
-            "<db-port> <user-name> <password> <isd-gate-url> <isd-admin-username> <isd-admin-password> <sapor-host-url> <audit-service-url> <redis/sql> <redis-host/sql-host> <redis-port/sql-username> <redis-password/sql-password> <migration-flag> <isd-platform-url>")
+            "Please pass valid 25 arguments <platform_db-name> <platform_host> <oes-db-name> <oes-db-host> <autopilot-db-name> <autopilot-db-host> <audit_db-name> <audit-db-host> <visibility_db-name> <visibility-db-host> "
+            "<db-port> <user-name> <password> <isd-gate-url> <isd-admin-username> <isd-admin-password> <sapor-host-url> <audit-service-url> <redis/sql/postgres> <redis-host/sql-host/postgres-host> <redis-port/sql-username/postgres-username> <redis-password/sql-password/postgres-password> <migration-flag> <isd-platform-url>  <sql-port/postgres-port>")
```

Comment:
@sudhakaropsmx we need to have uniformity here while dealing with params (otherwise it will be real tricky and error prone during automation / running-thourgh -job)- 
<redis-port/sql-username/postgres-username> -> should be :  <redis-port/sql-port/postgres-port>
Next param can be : <dummy/sql-username/postgres-username>
and then : <redis-password/sql-password/postgres-password>

And, our isntruction should be very clear that for redis, "dummy" value to be passed; we can't skip the input param